web_src/less/_base.less

```diff
@@ -2121,8 +2121,9 @@ a.ui.label:hover {
   font: 12px var(--fonts-monospace);
   white-space: pre-wrap;
   word-break: break-all;
-  overflow-wrap: break-word;
-  word-wrap: break-word;
+  overflow-wrap: anywhere;
+  /* stylelint-disable-next-line declaration-property-value-no-unknown */
+  word-wrap: anywhere; // Firefox respects word-wrap more than overflow-wrap
```

Comment:
`word-wrap` is legacy a per MDN:

> The property was originally a nonstandard and unprefixed Microsoft extension called word-wrap, and was implemented by most browsers with the same name. It has since been renamed to overflow-wrap, with word-wrap being an alias.